Jenkinsfile

```diff
@@ -91,7 +91,8 @@ pipeline {
       steps {
          sh 'git clone https://github.com/NVIDIA/Megatron-LM.git && \
              cd Megatron-LM && \
-             git checkout 98da3792f53c80ac9e865eab49a6fa5ccc293d22 && \
+             git checkout 81dab6067a0ef4635270b7e6e56bdf79ccfd7731 && \
+             touch megatron/core/models/retro/__init__.py && \
              pip install .'
```

Comment:
There's some issues with installing mcore recently in jenkins. See #8755 for more details.